### ISSUE TITLE
top level domain gives cert error

### ISSUE THREAD

**Issue by pwright** [Apicurio/apicurio.github.io#66]
Navigating to apicur.io (no www) gives the following cert issue

 Firefox does not trust this site because it uses a certificate that is not valid for apicur.io. The certificate is only valid for the following names: *.github.com, github.com

**Comment by EricWittmann** [comment]
Yeah I've noticed this as well.  I don't remember how the cert is created.  Not sure if this problem is high enough on the priority list to dig into, unless someone has some ideas! :)

**Comment by pwright** [comment]
I'd guess there's a CNAME dns record set up for www for Apicurio.github.io
but I think you also need A dns records to resolve as per https://docs.github.com/en/pages/configuring-a-custom-domain-for-your-github-pages-site/managing-a-custom-domain-for-your-github-pages-site#configuring-an-apex-domain
